doc/rippled-example.cfg

```diff
@@ -606,30 +606,49 @@
 #   Partial pathnames will be considered relative to the location of
 #   the rippled.cfg file.
 #
-#   [node_db]       Settings for the NodeDB (required)
-#   [temp_db]       Settings for the look-aside temporary db (optional)
-#   [import_db]     Settings for performing a one-time import (optional)
+#   [node_db]               Settings for the Node Database (required)
 #
 #   Format (without spaces):
-#       One or more lines of key / value pairs:
+#       One or more lines of case-insensitive key / value pairs:
 #       <key> '=' <value>
 #       ...
 #
-#   Examples:
-#       type=rocksdb
-#       path=db/rocksdb
+#   Example:
+#       type=nudb
+#       path=db/nudb
+#
+#   The "type" field must be present and controls the choice of backend:
+#
+#   type = NuDB
+#
+#       NuDB is high performance, but only works with Solid State (SSD)
+#       Drives or installations that have higher IOPS. It maintains its
+#       high speed regardless of the amount of history stored. Online
+#       delete may be selected, but is not required. NuDB is available
+#       on all platforms that rippled runs on.
+#
+#   type = RocksDB
+#
+#       This can work with spinning disks or installations that have
+#       lower IOPS (read/write throughput). It is not suitable for
```

Comment:
If you need to define IOPS, you should do it the first time you mention it (above).